config/prod.exs

```diff
@@ -4,7 +4,8 @@ import Config
 
 config :orbit,
   # loaded at compile time
-  release: System.get_env("RELEASE")
+  release: System.get_env("RELEASE"),
+  force_https?: true
```

Comment:
**suggestion:** I'd have a slight preference for setting `force_https?: true` in `config.exs` and only setting it to `false` explicitly in `dev.exs` just to be a bit safer.